Script.php

```diff
@@ -827,7 +827,13 @@ function checksize($file)
      if ($con->query($sql1) === TRUE) {
      }
      
-    
+
+    // When $group_id is not properly initialized, use integer 0 as its value.
+    // This temporarily fixed the "Students unable to submit assignment after a recent change" bug at http://118.25.96.118/bugzilla/show_bug.cgi?id=65
+    if (trim($group_id) === '') {
```

Comment:
@spm2020spring 
This worked, the Quiz 5 was submitted successfully. What does trim() do?

**Ashly**

Reply:
> @spm2020spring
> This worked, the Quiz 5 was submitted successfully. What does trim() do?
> 
> **Ashly**

@Teecloudy 

The function trim() removes all whitespaces.  

Therefore,  trim($group_id) will always return an empty string `''` if $group_id does not contain non-whitespace characters.  

**Hui**